### DISCUSSION TITLE
Draw.io/Diagrams.net self hosting

### DISCUSSION THREAD

**Discussion by winkelement** [outline/outline#4554] [Self Hosting] [answered]
In `settings/integrations/self-hosted` there is an option to add the url of a self-hosted instance of draw.io. It states "Add your self-hosted draw.io installation url here to enable automatic embedding of diagrams within documents." yet adding my self-hosted instance (tested working) seems to have no effect. How do i enable this "automatic embedding within documents"?

**Comment by tommoor** [comment]
Embeds can be disabled in Settings -> Security and within a specific document – double check that's not the case by embedding a YouTube video or something.

**Comment by winkelement** [comment, marked as answer]
Thank you I figured it out (I misunderstood the level of integration into outline).
